src/Helper/Escaper/AbstractHelper.php

```diff
@@ -124,7 +127,7 @@ public function setEncoding($encoding)
     /**
      * Get the encoding to use for escape operations
      *
-     * @deprecated Will be removed in 3.0.
+     * @deprecated Encoding is now considered a configuration item and not relevant in a template context
```

Comment:
The information of the version was good, so it should be kept.

Suggestion:

```suggestion
     * @deprecated since 2.20.0, this method will be removed in version 3.0.0 of this component.
                   Encoding is now considered a configuration item and not relevant in a template context
```